src/Bridges/Constraint/map.jl

```diff
@@ -0,0 +1,188 @@
+struct Map <: AbstractDict{MOI.VariableIndex, AbstractBridge}
+    # Constraint Index of bridged constraint -> Bridge.
+    # It is set to `nothing` when the constraint is deleted.
+    bridges::Vector{Union{Nothing, AbstractBridge}}
```

Comment:
Potentially a use-case for `CleverDict`?

Reply:
Yes, indeed, still not convinced that `Dict` is faster than `Vector{Union{Nothing, T}}` but early optimization is ...